diff --git a/dist/framework/waku.js b/dist/framework/waku.js
index 84e5f2d2ec29bd8ba3316a0c95fe529870b0e651..8c9e1ba2ca90c6f620511309a70f3ac7e63319b2 100644
--- a/dist/framework/waku.js
+++ b/dist/framework/waku.js
@@ -40,7 +40,7 @@ var framework = {
     );
   },
   Link({ href, prefetch: _prefetch, ...props }) {
-    return /* @__PURE__ */ jsx(WakuLink, { to: href, ...props, children: props.children });
+    return /* @__PURE__ */ jsx(WakuLink, { to: href, ...props, unstable_prefetchOnView: true ,children: props.children });
   }
 };
 function WakuProvider({ children }) {
